.buildkite/pipeline.yml

```diff
@@ -78,21 +78,21 @@ steps:
     command:
       - make -C runtime integ-test FICD_DM_POOL=build_${BUILDKITE_BUILD_NUMBER}_runtime
 
-  - label: ":running: snapshotter isolated tests"
-    agents:
-      queue: "${BUILDKITE_AGENT_META_DATA_QUEUE:-default}"
-      distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
-      hostname: "${BUILDKITE_AGENT_META_DATA_HOSTNAME}"
-    env:
-      DOCKER_IMAGE_TAG: "$BUILDKITE_BUILD_NUMBER"
-      NUMBER_OF_VMS: 10
-      EXTRAGOARGS: "-v -count=1 -race -timeout 3m"
-      FICD_DM_VOLUME_GROUP: fcci-vg
-    artifact_paths:
-      - "snapshotter/logs/*"
-    command:
-      - make -C snapshotter integ-test FICD_DM_POOL=build_${BUILDKITE_BUILD_NUMBER}_snapshotter
-    timeout_in_minutes: 10
+  # - label: ":running: snapshotter isolated tests"
```

Comment:
Do we intend to enable these tests in the future? Please cut a issue so that we can track it once this PR is merged.

Reply:
Probably not. The snapshotter is no longer needed. But i just commented this block in case we'd like to revoke this decision later.